packages/orbit-components/src/utils/common/index.ts

```diff
@@ -1,15 +1,25 @@
-import type { FlattenSimpleInterpolation } from "styled-components";
+import type {
+  FlattenInterpolation,
+  FlattenSimpleInterpolation,
+  ThemeProps,
+} from "styled-components";
 import { css } from "styled-components";
 
 import type { ObjectProperty } from "../../common/types";
+import defaultTheme from "../../defaultTheme";
 
 export const firstToUpper = (str: string): string => str.charAt(0).toUpperCase() + str.slice(1);
 
-export const defaultFocus = (): FlattenSimpleInterpolation => css`
-  outline: 2px solid Highlight;
-  outline: 2px solid -webkit-focus-ring-color;
+export const defaultFocus = (): FlattenInterpolation<ThemeProps<typeof defaultTheme>> => css`
+  ${({ theme }) => css`
+    outline: 2px solid ${theme.orbit.paletteBlueNormal};
+  `};
 `;
 
+defaultFocus.defaultProps = {
+  theme: defaultTheme,
+};
```

Comment:
Do we need this? `defaultFocus` is not a component 🤔 weird calling it defaultProps…

Reply:
If anything it should the component's responsibility to have theme as default prop (like you added in StyledStepperButton

Reply:
yes, we need, defaultFocus need access to theme object

Reply:
right, forgot to remove that, when pipeline was failing and i was not sure where exactly it can't access theme object, turned out that it's in Stepper

Reply:
But it's not a component and we're defining default props 🤯 can't it be default value for the argument instead? It's non-blocking though